deployability/modules/testing/tests/test_agent/test_uninstall.py

```diff
@@ -1,38 +1,46 @@
 import grp
 import pwd
-
-
-from ..helpers import constants
-
-
-# @pytest.fixture(scope='module', autouse=True)
-# def uninstall_wazuh():
-#     service = utils.get_service()
-#     daemon_name = 'wazuh-agent' if service == 'agent' else 'wazuh-manager'
-#     utils.run_command('apt-get', ['purge', daemon_name, '-y'])
-
+import pytest
+
+from ..helpers import constants, utils
+from ..helpers.uninstaller import WazuhUninstaller
+from ..helpers.checkfiles import CheckFile
+from ..helpers.hostinformation import HostInformation
+
+
+@pytest.fixture
+def wazuh_params(request):
+    return {
+        'wazuh_version': request.config.getoption('--wazuh_version'),
+        'wazuh_revision': request.config.getoption('--wazuh_revision'),
+    }
+
+def test_uninstallation(wazuh_params):
+    hostinfo= HostInformation()
+    uninstall_args = (
+        hostinfo.get_os_type(),
+        wazuh_params['wazuh_version'],
+        wazuh_params['wazuh_revision'],
+        hostinfo.get_linux_distribution()
+    )
+    checkfile= CheckFile()
+    wazuh_uninstaller= WazuhUninstaller(*uninstall_args)
+    result = checkfile.perform_action_and_scan(lambda: wazuh_uninstaller.uninstall_agent())
+
+    assert all('wazuh' in path or 'ossec' in path for path in result['removed'])
+    assert not any('wazuh' in path or 'ossec' in path for path in result['added'])
 
 def test_wazuh_user():
     all_users = [x[0] for x in pwd.getpwall()]
     assert constants.WAZUH_USER not in all_users, "Wazuh user found."
 
-
 def test_wazuh_group():
     all_groups = [x[0] for x in grp.getgrall()]
     assert constants.WAZUH_GROUP not in all_groups, "Wazuh group found."
 
+def test_process_not_running():
+    assert not utils.is_process_alive('wazuh'), 'Wazuh process is running.'
 
-def test_wazuh_configuration_dir():
-    assert not constants.CONFIGURATIONS_DIR.exists(), "Configuration directory found."
-
-
-def test_wazuh_configuration():
-    assert not constants.WAZUH_CONF.exists(), "Configuration file found."
-
-
-def test_wazuh_binaries_dir():
-    assert not constants.BINARIES_DIR.exists(), "Binaries directory found."
-
-
-def test_wazuh_control():
-    assert not constants.WAZUH_CONTROL.exists(), "Wazuh control binary found."
+def test_config_is_not_maintained():
+    assert not constants.WAZUH_CONF.exists(), "Wazuh config file waz found."
+    assert not constants.LOCAL_INTERNAL_OPTIONS.exists(), "Local internal options file was found."
```

Comment:
Add newline at EOF
```suggestion
    assert not constants.LOCAL_INTERNAL_OPTIONS.exists(), "Local internal options file was found."

```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4998/commits/db29c08420c01890d8ca7fedddeb25b94ab07fc3